setupext/test_java.py

```diff
@@ -53,7 +53,7 @@ def compileJava():
     srcs = glob.glob('test/harness/jpype/**/*.java', recursive=True)
     srcs.extend(glob.glob('test/harness/org/**/*.java', recursive=True))
     exports = ""
-    if version > 7:
+    if version == 8:
```

Comment:
Looks correct, but unrelated.

Reply:
It was an old bug that cropped up because when I did a full clean there are two copies of a file.  One for java 8 and one for 9.   I can make another PR but it seem like a lot of work to maje that change seperately then .merge it back.

Reply:
I keep hoping Java 8 will retire as it has a some reflection differences.  But it wont retire until 2026!